src/sentry/processing/realtime_metrics/base.py

```diff
@@ -36,3 +40,78 @@ def increment_project_event_counter(self, project_id: int, timestamp: float) ->
             pipeline.set(key, 0, nx=True, px=self._counter_ttl)
             pipeline.incr(key)
             pipeline.execute()
+
+    # TODO: do these killswitch helpers belong here, or in a different class? LowPriorityQueueStore?
+    # This probably needs locking
+    def get_lpq_projects(self) -> Set[int]:
```

Comment:
Do we use this?  And will we need to use it?  I was more expecting that we'd need a `is_project_lqp(self, project_id: int) -> bool`. That's O(1), this is O(n). plus you get rid of some annoying typing issues of this `_to_int()` function.

Reply:
conversation was continued and resolved in https://github.com/getsentry/sentry/pull/28714#discussion_r713891555 as that PR uses this helper.

tl;dr it's the simplest solution right now to get the info we need, but it could definitely be improved if this ends up being a source of performance problems.